src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ASN1.cs

```diff
@@ -61,6 +61,22 @@ internal static partial class Crypto
         [DllImport(Libraries.CryptoNative)]
         internal static extern void Asn1StringFree(IntPtr o);
 
+        internal static string GetOidValue(SafeSharedAsn1ObjectHandle asn1Object)
+        {
+            Debug.Assert(asn1Object != null);
+
+            bool added = false;
+            asn1Object.DangerousAddRef(ref added);
+            try
+            {
+                return GetOidValue(asn1Object.DangerousGetHandle());
```

Comment:
Just that it was written before I'd started partying with SafeInteriorHandles.  I want to clean up the rest of the usages, but didn't want to conflict with Eric's changes for shimming; so kept this as a style adapter.